File: console.app/EduHome/EduHome/Entities/Course.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EduHome.Entities
{
    internal class Course
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Starts { get; set; }

        public int Duration { get; set; }

        public int ClassDuration { get; set; }

        public string Language { get; set; }

        public int Students { get; set; }

        public int Price { get; set; }

        public string Image {  get; set; }

    }
}

File: Front/courses/course.js
function loadProductById(id) {
    fetch(`http://localhost:5000/courses/${id}`)
        .then(response => response.json())
        .then(product => {
            document.getElementById("name").innerHTML = product.name;
            document.getElementById("starts").innerHTML = product.starts;
            document.getElementById("duration").innerHTML = `${product.duration} Month`;
            document.getElementById("cDuration").innerHTML = `${product.classDuration} Hours`;
            document.getElementById("language").innerHTML = product.language;
            document.getElementById("students").innerHTML = product.students;
            document.getElementById("fee").innerHTML = `COURSE FEE $${product.price}`;
            document.getElementById("bigImg").src = product.image;
            document.getElementById("bigImg").style.width = '821px';
            
        })
        .catch(error => {
            console.error('Error fetching product data:', error);
        });
}


let redirectedId = window.location.hash.substring(1);


loadProductById(redirectedId);


